### PR TITLE
sorting broken into individual components

### PR DESCRIPTION
**Pull request by Jesuitman**
What I did: Broke up the 2 sort components into their own components which are nestled into the modal directory
Did this break anything?
- [x] No
- [ ]  Yes
Type of change
- [ ] New feature (non-breaking change which adds functionality)
- [ ]  Styling
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.
Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing
What's next?

### CODE REVIEW

File: src/ParkChecklist/ParkChecklist.js [hunk 1]
```diff
@@ -157,57 +147,17 @@ const ParkChecklist = ({ apiLink }) => {
...
-                      </label>
-                    ))}
-                  </div>
-                </div>
-              </label>
+              <StateSort
+                states={states}
+                selectedStateParks={selectedStateParks}
+                handleStateCheckboxChange={handleStateCheckboxChange}
+              />
```
Comment (JoeyStewart) [inline src/ParkChecklist/ParkChecklist.js:154]: Much cleaner. Thank you for doing this!

**Review by JoeyStewart — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -89,3 +89,12 @@ Joey -  It was our first time working with a backend team, we needed to discuss
 - user wants to be able to get to the details of a park page by clicking on it somewhere on the checklist view (not being forced to go through state view)
 - user wants a wishlist or a planning to visit type of page
 
+
```
[the remark this answers is not in the document]
Comment (JoeyStewart) [reply README.md:92]: Looks like those were notes from the user tests. We just can probably delete them and use that space for our next test.

File: src/ParkChecklist/ParkChecklist.js [hunk 3]
```diff
@@ -299,7 +249,7 @@ const ParkChecklist = ({ apiLink }) => {
       path: `/login`,
       element: (
         <div>
-          <Login loggedIn={loggedIn} setLoggedIn={setLoggedIn}/>
+          <Login loggedIn={loggedIn} setLoggedIn={setLoggedIn} />
```
Comment (tenthwalker) [inline src/ParkChecklist/ParkChecklist.js:252]: In our component refactor branch, we will want to pull this out to the App level with the rest of the routing

File: src/ParkChecklist/ParkChecklist.js (hunk 1 above)
Comment (tenthwalker) [reply src/ParkChecklist/ParkChecklist.js:154]: second this, it'll be much easier to parse through this file now!

**Review by tenthwalker — APPROVED** [review]
Thanks for breaking those out, this is a really helpful step towards better developer empathy in our code base.

**Pull request merged**